src/Riok.Mapperly/Descriptors/DescriptorBuilder.cs

```diff
@@ -123,9 +127,29 @@ private void ExtractUserMappings()
                 userMapping.TargetType
             );
 
+            if (userMapping.Method is { IsStatic: true, IsPartialDefinition: true })
```

Comment:
Is the partial definition part really required? IMO also user implemented mapping methods need to be considered, don't they?

Reply:
Ah you are correct I didn't thought about that one.